index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Project</title>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+    <h1 class = "page-header">Hello Microverse!</h1>
```

Comment:
- [OPTIONAL]  I suggest not adding blank lines, spaces, or indentations without a reason To follow best practices for HTML & CSS. Kindly, Use space-less equal signs.

> For example
-  use `<h1 class="page-header">` instead of `<h1 class = "page-header">`

Reply:
Thank you very much for the review and the suggestion